### ISSUE TITLE
238 | Product of Array Except Self | Python

### ISSUE THREAD

**Issue by Jahnavi-Jonnalagadda** [noisefilter19/LeetCode_Algorithms#654]
Problem name: Product of Array Except Self
Difficulty: Medium
Language: Python

**Comment by Jahnavi-Jonnalagadda** [comment]
Sir, could you please assign this issue to me?

**Comment by Jahnavi-Jonnalagadda** [comment]
@noisefilter19, it is giving an error while cloning the repo. And says invalid path for "product of last k numbers.py". Could you please look into this issue?

**Comment by noisefilter19** [comment]
Check now

**Comment by Jahnavi-Jonnalagadda** [comment]
No sorry, its the same @noisefilter19. This is the error message - "error: invalid path 'Python/1352 | Product of the last k numbers.py' "

**Comment by noisefilter19** [comment]
Nope. There is no such file as of now. Take a pull from repo again and try
